packages/glimmer-runtime/lib/dom/change-lists.ts

```diff
@@ -126,6 +134,27 @@ export const InputValuePropertyChangeList = {
   }
 };
 
+export const TextAreaValueChangeList = {
+  setAttribute(dom: DOMHelper, element: Element, attr: string, value: Opaque) {
+    let textArea = <HTMLTextAreaElement>element;
+    let normalizedValue = normalizeTextValue(value);
+
+    if(textArea.childNodes.length === 0) {
+      let textNode = dom.createTextNode(normalizedValue);
+      textArea.appendChild(textNode);
+    } else {
+      let currentValue = textArea.value;
```

Comment:
Can this section be safely moved to the `updateAttribute` method?

Something like:

```js
export const TextAreaValueChangeList = {
  setAttribute(dom: DOMHelper, element: Element, attr: string, value: Opaque) {
    let textArea = <HTMLTextAreaElement>element;
    let normalizedValue = normalizeTextValue(value);
    let textNode = dom.createTextNode(normalizedValue);
    textArea.appendChild(textNode);
  },

  updateAttribute(dom: DOMHelper, element: Element, attr: string, value: Opaque) {
    let textArea = <HTMLTextAreaElement>element;
    let currentValue = textArea.value;
    if (currentValue !== normalizedValue) {
      textArea.value = normalizedValue;
    }
  }
};
```

Are there cases that ^ wouldn't work for?